src/client/components/forms/userCollection.js

```diff
@@ -160,6 +172,24 @@ class UserCollectionForm extends React.Component {
 		return (
 			<div>
 				<h1>Create your collection</h1>
+				<DeleteCollectionModal
+					closeDeleteModalCallback={this.closeDeleteModalCallback}
+					collection={this.props.collection}
+					show={this.state.showModal}
+				/>
+				{
+					this.props.collection.id ?
+						<Button
+							bsSize="small"
+							bsStyle="danger"
+							className="pull-right"
+							type="button"
+							onClick={this.handleShowDeleteModal}
+						>
+							<FontAwesomeIcon icon="times"/>&nbsp;Delete this collection
```

Comment:
I'd say the trash icon would be better here for consistency, and I'd remove `this` to match the grammar of the other button

Reply:
I've also seen before the use of elipsis (`…`) on delete buttons (and often in menu items) to suggest there is a further action, or at least that it's not an instant deletion. What do you think about that?